src/Polymake.jl

```diff
@@ -55,9 +55,12 @@ function __init__()
         prepare_env()
     end
 
+    show_banner = isinteractive() &&
```

Comment:
Since we're already changing this why don't we use `parentmodule`?
in general have a look at https://github.com/JuliaLang/julia/blob/master/base/reflection.jl;

`Base.package_locks` which doesn't do what we want